disperser/cmd/controller/flags/flags.go

```diff
@@ -322,6 +322,12 @@ var (
 		Value:    30 * time.Second,
 		EnvVar:   common.PrefixEnvVar(envVarPrefix, "PAYMENT_VAULT_UPDATE_INTERVAL"),
 	}
+	DisperserIDFlag = cli.Uint64Flag{
```

Comment:
Does this actually need to be a CLI flag?

Since the disperser will have its own pubkey, it seems like it should be able to just fetch its `DisperserID` from the `DisperserRegistry`. That way, we avoid adding an additional config field that can get out of sync

Reply:
`DisperserRegistry` does not currently provide an `Address->ID` method.

Reply:
I forgot we are missing that- it's actually part of my [draft](https://linear.app/eigenlabs/issue/EGDA-2236/modify-disperserregistry-to-support-permissionless-dispersal) of changes we should make for permissionless dispersal.

I agree the strategy you used makes sense in the short term- we can just remove this config requirement once the contract has been updated with the necessary functions